kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/consumer/streamable/ConsumerPositionsUtilIntegrationTest.java

```diff
@@ -95,9 +96,9 @@ void positionsTest() {
         String topic = "testPositionsUtil";
 
         Consumer<?, ?> testConsumer = consumerFactory.createConsumer(null);
-        List<String> topics = Collections.singletonList(topic);
-        assertTrue(ConsumerPositionsUtil.getHeadPositions(testConsumer, topics).isEmpty());
-        assertTrue(ConsumerPositionsUtil.getPositionsBasedOnTime(testConsumer, topics, Instant.now()).isEmpty());
+        ListKafkaSubscriber subscriber = new ListKafkaSubscriber(Collections.singletonList(topic));
```

Comment:
I think it would be nice to have a test case validating the pattern subscriber to work with the `ConsumerPositionsUtils` too.

Reply:
Done, let me know your thoughts on the approach.